tools/generator/C/include_v1.0/pprzlink_device.h

```diff
@@ -42,9 +42,10 @@
  * they are used to cast the real functions with the correct type
  * to store in the device structure
  */
-typedef int (*check_free_space_t)(void *, uint8_t);
-typedef void (*put_byte_t)(void *, uint8_t);
-typedef void (*send_message_t)(void *);
+typedef int (*check_free_space_t)(void *, long *, uint8_t);
+typedef void (*put_byte_t)(void *, long, uint8_t);
+typedef void (*put_buffer_t)(void *, long, uint8_t, uint8_t *);
```

Comment:
It was this way because put_bytes_t in pprzlink_transport API is giving len first. Which one should we keep ?

I agree for the size.